diff --git a/src/source/test/cintltst/creststn.c b/src/source/test/cintltst/creststn.c
index a19723b..bc1b0ae 100644
--- a/src/source/test/cintltst/creststn.c
+++ b/src/source/test/cintltst/creststn.c
@@ -951,7 +951,13 @@ static void TestBinaryCollationData(){
     }
     status=U_ZERO_ERROR;
     coll = ures_getByKey(teRes, "collations", coll, &status);
+    if(U_FAILURE(status)){
+        log_err("ODADebug: ERROR: ures_getByKey(teRes, \"collations\", coll, &status)");
+    }
     coll = ures_getByKey(coll, "standard", coll, &status);
+    if(U_FAILURE(status)){
+        log_err("ODADebug: ERROR: ures_getByKey(coll, \"standard\", coll, &status)");
+    }
     if(U_SUCCESS(status)){
         CONFIRM_ErrorCode(status, U_ZERO_ERROR);
         CONFIRM_INT_EQ(ures_getType(coll), URES_TABLE);
