diff --git a/src/source/runConfigureICU b/src/source/runConfigureICU
index 77df3a0..b1d6e24 100755
--- a/src/source/runConfigureICU
+++ b/src/source/runConfigureICU
@@ -269,6 +269,39 @@ case $platform in
         DEBUG_CXXFLAGS='-FS -Zi -MDd'
         DEBUG_LDFLAGS='-DEBUG'
         ;;
+    Cygwin/MSVC_ClangCL)
+        THE_OS="Windows with Cygwin"
+        THE_COMP="Microsoft Visual C++ ClangCL (dynamic runtime)"
+        CC=clang-cl; export CC
+        CXX=clang-cl; export CXX
+        RELEASE_CFLAGS='-Gy -MD'
+        RELEASE_CXXFLAGS='-Gy -MD'
+        DEBUG_CFLAGS='-FS -Zi -MDd'
+        DEBUG_CXXFLAGS='-FS -Zi -MDd'
+        DEBUG_LDFLAGS='-DEBUG'
+        ;;
+    Cygwin/MSVC_ClangCL_MT)
+        THE_OS="Windows with Cygwin"
+        THE_COMP="Microsoft Visual C++ ClangCL (static runtime)"
+        CC=clang-cl; export CC
+        CXX=clang-cl; export CXX
+        RELEASE_CFLAGS='-Gy -MT'
+        RELEASE_CXXFLAGS='-Gy -MT'
+        DEBUG_CFLAGS='-FS -Zi -MTd'
+        DEBUG_CXXFLAGS='-FS -Zi -MTd'
+        DEBUG_LDFLAGS='-DEBUG'
+        ;;
+    Cygwin/MSVC_MT)
+        THE_OS="Windows with Cygwin"
+        THE_COMP="Microsoft Visual C++ (static runtime)"
+        CC=cl; export CC
+        CXX=cl; export CXX
+        RELEASE_CFLAGS='-Gy -MT'
+        RELEASE_CXXFLAGS='-Gy -MT'
+        DEBUG_CFLAGS='-FS -Zi -MTd'
+        DEBUG_CXXFLAGS='-FS -Zi -MTd'
+        DEBUG_LDFLAGS='-DEBUG'
+        ;;
     Cygwin/MSVC2005)
         THE_OS="Windows with Cygwin"
         THE_COMP="Microsoft Visual C++ 2005"
