diff --git a/src/source/configure b/src/source/configure
index 9ef981d..ca737fd 100755
--- a/src/source/configure
+++ b/src/source/configure
@@ -9138,7 +9138,7 @@ else
       touch data/rules.mk
     fi
     echo "Spawning Python to generate test/testdata/rules.mk..."
-    PYTHONPATH="$srcdir/test/testdata:$srcdir/data" $PYTHON -m buildtool \
+    PYTHONPATH="$srcdir/test/testdata;$srcdir/data" $PYTHON -m buildtool \
         --mode gnumake \
         --seqmode parallel \
         --src_dir "$srcdir/test/testdata" \
diff --git a/src/source/configure.ac b/src/source/configure.ac
index 8cd270b..931158a 100644
--- a/src/source/configure.ac
+++ b/src/source/configure.ac
@@ -1414,7 +1414,7 @@ else
       touch data/rules.mk
     fi
     echo "Spawning Python to generate test/testdata/rules.mk..."
-    PYTHONPATH="$srcdir/test/testdata:$srcdir/data" $PYTHON -m buildtool \
+    PYTHONPATH="$srcdir/test/testdata;$srcdir/data" $PYTHON -m buildtool \
         --mode gnumake \
         --seqmode parallel \
         --src_dir "$srcdir/test/testdata" \
