diff --git a/remove_rebuild_rules.patch b/remove_rebuild_rules.patch
index dd8d9b0..e69de29 100644
--- a/remove_rebuild_rules.patch
+++ b/remove_rebuild_rules.patch
@@ -1,45 +0,0 @@
-diff --git a/src/source/configure b/src/source/configure
-index 9ef981d..f3a6e0e 100755
---- a/src/source/configure
-+++ b/src/source/configure
-@@ -9115,38 +9115,8 @@ $as_echo "$as_me: WARNING: unrecognized options: $ac_unrecognized_opts" >&2;}
- fi
- 
- 
--if test -z "$PYTHON";
--then
--    echo "" > data/rules.mk
--    echo "" > test/testdata/rules.mk
--else
--    if test -f "$srcdir/data/locales/root.txt";
--    then
--        echo "Spawning Python to generate data/rules.mk..."
--        PYTHONPATH="$srcdir/data" $PYTHON -m buildtool \
--            --mode gnumake \
--            --seqmode parallel \
--            --src_dir "$srcdir/data" \
--            --filter_file "$ICU_DATA_FILTER_FILE" \
--            $ICU_DATA_BUILDTOOL_OPTS \
--            > data/rules.mk
--        if test "$?" != "0"; then
--            as_fn_error $? "Python failed to run; see above error." "$LINENO" 5
--        fi
--    else
--      echo "Not rebuilding data/rules.mk, assuming prebuilt data in data/in"
--      touch data/rules.mk
--    fi
--    echo "Spawning Python to generate test/testdata/rules.mk..."
--    PYTHONPATH="$srcdir/test/testdata:$srcdir/data" $PYTHON -m buildtool \
--        --mode gnumake \
--        --seqmode parallel \
--        --src_dir "$srcdir/test/testdata" \
--        > test/testdata/rules.mk
--    if test "$?" != "0"; then
--        as_fn_error $? "Python failed to run; see above error." "$LINENO" 5
--    fi
--fi
-+echo "" > data/rules.mk
-+echo "" > test/testdata/rules.mk
- 
- echo
- echo "ICU for C/C++ $VERSION is ready to be built."
diff --git a/src/source/configure b/src/source/configure
index 9ef981d..f3a6e0e 100755
--- a/src/source/configure
+++ b/src/source/configure
@@ -9115,38 +9115,8 @@ $as_echo "$as_me: WARNING: unrecognized options: $ac_unrecognized_opts" >&2;}
 fi
 
 
-if test -z "$PYTHON";
-then
-    echo "" > data/rules.mk
-    echo "" > test/testdata/rules.mk
-else
-    if test -f "$srcdir/data/locales/root.txt";
-    then
-        echo "Spawning Python to generate data/rules.mk..."
-        PYTHONPATH="$srcdir/data" $PYTHON -m buildtool \
-            --mode gnumake \
-            --seqmode parallel \
-            --src_dir "$srcdir/data" \
-            --filter_file "$ICU_DATA_FILTER_FILE" \
-            $ICU_DATA_BUILDTOOL_OPTS \
-            > data/rules.mk
-        if test "$?" != "0"; then
-            as_fn_error $? "Python failed to run; see above error." "$LINENO" 5
-        fi
-    else
-      echo "Not rebuilding data/rules.mk, assuming prebuilt data in data/in"
-      touch data/rules.mk
-    fi
-    echo "Spawning Python to generate test/testdata/rules.mk..."
-    PYTHONPATH="$srcdir/test/testdata:$srcdir/data" $PYTHON -m buildtool \
-        --mode gnumake \
-        --seqmode parallel \
-        --src_dir "$srcdir/test/testdata" \
-        > test/testdata/rules.mk
-    if test "$?" != "0"; then
-        as_fn_error $? "Python failed to run; see above error." "$LINENO" 5
-    fi
-fi
+echo "" > data/rules.mk
+echo "" > test/testdata/rules.mk
 
 echo
 echo "ICU for C/C++ $VERSION is ready to be built."
diff --git a/src/source/configure.ac b/src/source/configure.ac
index 8cd270b..6bccd4a 100644
--- a/src/source/configure.ac
+++ b/src/source/configure.ac
@@ -1391,38 +1391,8 @@ AC_CONFIG_FILES([icudefs.mk \
 		samples/cal/Makefile samples/layout/Makefile])
 AC_OUTPUT
 
-if test -z "$PYTHON";
-then
-    echo "" > data/rules.mk
-    echo "" > test/testdata/rules.mk
-else
-    if test -f "$srcdir/data/locales/root.txt";
-    then
-        echo "Spawning Python to generate data/rules.mk..."
-        PYTHONPATH="$srcdir/data" $PYTHON -m buildtool \
-            --mode gnumake \
-            --seqmode parallel \
-            --src_dir "$srcdir/data" \
-            --filter_file "$ICU_DATA_FILTER_FILE" \
-            $ICU_DATA_BUILDTOOL_OPTS \
-            > data/rules.mk
-        if test "$?" != "0"; then
-            AC_MSG_ERROR(Python failed to run; see above error.)
-        fi
-    else
-      echo "Not rebuilding data/rules.mk, assuming prebuilt data in data/in"
-      touch data/rules.mk
-    fi
-    echo "Spawning Python to generate test/testdata/rules.mk..."
-    PYTHONPATH="$srcdir/test/testdata:$srcdir/data" $PYTHON -m buildtool \
-        --mode gnumake \
-        --seqmode parallel \
-        --src_dir "$srcdir/test/testdata" \
-        > test/testdata/rules.mk
-    if test "$?" != "0"; then
-        AC_MSG_ERROR(Python failed to run; see above error.)
-    fi
-fi
+echo "" > data/rules.mk
+echo "" > test/testdata/rules.mk
 
 echo
 echo "ICU for C/C++ $VERSION is ready to be built."
